### DISCUSSION TITLE
抽到朱鸢最重要的是……

### DISCUSSION THREAD

**Discussion by canifish** [share121/inter-knot#1477] [其他]
![20240810181907](https://github.com/user-attachments/assets/fbc3dd0b-73bd-416c-9f01-9410204eb85f)
是强度吗？是升级妮可连搓招吗？都不是！当然是大辟谷😃😃😃

**Comment by Jiucheng258** [comment]
@ZhuYuan-148

**Comment by ZhuYuan-148** [comment]
诶？为什么要在后面拍呢…这位市民如果想合影的话直接问我就可以了
